flips/20220811-fungible-tokens-metadata.md

```diff
@@ -0,0 +1,210 @@
+
+### Drawbacks
+
+Main drawback could came from inefficient implementations of views from fungible token contracts.
+
+### Alternatives Considered
+
+Since the `MetadataViews` contract has been a successful solution for increasing NFT interoperability no alternatives have been considered for the FT. 
+
+### Performance Implications
+
+Due to the potentially dynamic nature of this proposal, the methods that return metadata views can be implemented in any way that the author sees fit which could lead to some non-performant implementations.
+
+### Dependencies
+
+This proposal builds on the existing FT interface defined [here](https://github.com/onflow/flow-ft).
+
+### Engineering Impact
+
+Adhering to this standard will require contract authors to implement a few new methods as part of their FT. They will also likely need to import standard types into their contracts.
+
+There may also need to be a committee, process, and code repository put in place for managing the standard metadata types.
+
+### Best Practices
+
+Fungible Token issuers should implement the `FTView` to achieve the as much ecosystem compatibility as possible.
+
+### Tutorials and Examples
+
+Guidelines on how to implement the new views will be added to the `ExampleToken.cdc` contract, and their used will be documented on the `flow-ft` repo docs.
+
+### Compatibility
+
+Fungible Token contract owners will need to update them to add implementations for the new methods.
+
+### User Impact
+
+This feature can be rolled out with no fear of changes to the user. However documentation should be provided on how they can change their contracts and add views.
+
+## Related Issues
+
+
+
+## Prior Art
+
+The already working NFT Metadata views [standard](https://github.com/onflow/flow/blob/master/flips/20210916-nft-metadata.md) has been used as the main source for this proposal.
+
+## Questions and Discussion Topics
+
+Main friction point will be if we want this standard to be in a new different contract of if we want it to be part of the existent `MetadataViews.cdc` contract.
+
+Are we missing any core views that should be mandatory from the beginning?
+
+Could there be some view, like serial, that differentiates tokens from the same `Vault` depending for instance of when they where minted? (think of different years coins with the same value) Would that make them NFT rather than FT? Would ever any developer want to do that? How could we keep track of a single balance which contains slightly different tokens? Would that introduce an unnecessary complication when transferring tokens? (of course it would). This could make sense along with the [streamlined tokens proposal new `FungibleToken.Balance` interface that exposes a `getBalance()` function rather than a field just in case the FT owner wants to compute the balance](https://forum.onflow.org/t/streamlined-token-standards-proposal/3075)
```

Comment:
I don't think we need to worry about this now. I think if people want that kind of view, they can submit a proposal after these initial views have been accepted and deployed.

Reply:
makes sense, getting rid of it!